WD-Firmware/rtos/rtosal/api_inc/rtosal_services_api.h

```diff
@@ -0,0 +1,66 @@
+/*
+* SPDX-License-Identifier: Apache-2.0
+* Copyright 2019 Western Digital Corporation or its affiliates.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http:*www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/**
+* @file   rtosal_services_api.h
+* @author Nati Rapaport
+* @date   18.09.2019
+* @brief  The file defines the RTOS AL services interfaces (minimal exposure)
+*/
+#ifndef __RTOSAL_SRV_API_H__
+#define __RTOSAL_SRV_API_H__
+
+/**
+* include files
+*/
+
+/**
+* definitions
+*/
+
+/**
+* macros
+*/
+
+/**
+* types
+*/
+
+/**
+* local prototypes
+*/
+
+/**
+* external prototypes
+*/
+
+/**
+* global variables
+*/
+
+/**
+* APIs
+*/
+
+
+/**
+* This function is invoked by the system timer interrupt
+*/
+void rtosalTick(void);
```

Comment:
1. this file is not needed
2. rtosalTick to be in rotsal.h -> is it loc include